tests/foreman/api/test_role.py

```diff
@@ -1402,7 +1402,7 @@ def test_positive_access_users_inside_org_admin_taxonomies(self, role_taxonomies
         except HTTPError as err:
             pytest.fail(str(err))
 
-    @pytest.mark.skip_if_open('BZ:1694199')
+    @pytest.mark.skip_if_open('BZ:1825698')
```

Comment:
Add the BZ in `:bz:` token in this tests docstring!

Reply:
Updated